metricflow/model/dbt_mapping_rules/dbt_metric_to_dimensions_rules.py

```diff
@@ -13,6 +13,40 @@
 from metricflow.model.validations.validator_helpers import ModelValidationResults, ValidationIssue, ValidationError
 
 
+DBT_COLUMN_TYPES_TO_DIMENSION_TYPES: Dict[str, DimensionType] = {
+    "DATE": DimensionType.TIME,
+    "TIMESTAMP_TZ": DimensionType.TIME,
+    "TIMESTAMP_NTZ": DimensionType.TIME,
+    "TEXT": DimensionType.CATEGORICAL,
+    "BOOLEAN": DimensionType.CATEGORICAL,
+    "NUMBER": DimensionType.CATEGORICAL,  # Measure?
+    "FLOAT": DimensionType.CATEGORICAL,  # Measure?
```

Comment:
Number and int could be anything. It's harder to come up with a situation where you'd want a floating point number as a dimension value, but eh. Metricflow allows it.